examples/with-chakra-ui/renderer/next.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  webpack: (config, { isServer }) => {
+    if (!isServer) {
+      config.target = 'electron-renderer'
+    }
+
+    return config
+  },
+}
```

Comment:
[MUST] Please add `images.unoptimized = true` in next.config.js if you use `Image` component, because nextron uses `next export` command when production build:

```
Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
  Possible solutions:
    - Use `next start` to run a server, which includes the Image Optimization API.
    - Configure `images.unoptimized = true` in `next.config.js` to disable the Image Optimization API.
  Read more: https://nextjs.org/docs/messages/export-image-api
```

```suggestion
module.exports = {
  webpack: (config, { isServer }) => {
    if (!isServer) {
      config.target = 'electron-renderer'
    }

    return config
  },
  images: {
    unoptimized: true,
  },
}
```

Reply:
Resolved